core/src/test/scala/tofu/examples/TofuRaiseExamples.scala

```diff
@@ -0,0 +1,84 @@
+package tofu.examples
```

Comment:
I suggest narrowing package namespace. Otherwise classes from different examples may clash.

Reply:
@catostrophe, can you be more specific?

Reply:
@oskin1 Sorry. This doesn't seem to be an issue. I meant classes in the `tofu.examples` package in different modules might clash.